payments_service/braintree/tests/test_views/test_webhook.py

```diff
@@ -147,18 +151,21 @@ def test_email_for_subscription_charge(self):
 
     def test_email_for_subscription_charge_failure(self):
         notice = subscription_notice(
+            'service-subscription',
             kind='subscription_charged_unsuccessfully'
         )
         self.solitude.braintree.webhook.post.return_value = notice
         self.post()
         self.assertEqual(len(mail.outbox), 1)
         self.assertEqual(mail.outbox[0].subject,
-                         'Brick: subscription charge failed')
+                         'Fake Subscription: subscription charge failed')
 
         msg = mail.outbox[0].body
-        assert 'Mozilla Concrete' in msg, 'Unexpected: {}'.format(msg)
+        assert 'Fake Service Provider' in msg, 'Unexpected: {}'.format(msg)
         assert 'Visa ending in 1234' in msg, 'Unexpected: {}'.format(msg)
-        assert 'Product       Brick' in msg, 'Unexpected: {}'.format(msg)
+        assert 'Product       Fake Subscription' in msg, (
```

Comment:
it's pattern matching within a plain text email (where the white space matters). Assertions are a bit ugly but I guess they are ok.